src/base/bittorrent/torrenthandle.cpp

```diff
@@ -59,6 +59,7 @@
 #include "peerinfo.h"
 #include "trackerentry.h"
 #include "torrenthandle.h"
+#include "gui/torrentcontentmodelitem.h"
```

Comment:
Its just demo but yeah probably should be in a separate file. What do you think would be best place to place to put the auto prioritization code?